Goobi/src/de/sub/goobi/export/download/ExportPdf.java

```diff
@@ -134,24 +132,31 @@ public boolean startExport(Prozess myProzess, String inZielVerzeichnis) throws I
 					if (contentServerUrl == null || contentServerUrl.length() == 0) {
 						contentServerUrl = myBasisUrl + "/cs/cs?action=pdf&images=";
 					}
-					String url = "";
 					FilenameFilter filter = new FileListFilter("\\d*\\.tif");
 					SafeFile imagesDir = new SafeFile(myProzess.getImagesTifDirectory(true));
 					SafeFile[] meta = imagesDir.listFiles(filter);
-					ArrayList<String> filenames = new ArrayList<String>();
+					int capacity = 19 + contentServerUrl.length() + myProzess.getTitel().length();
```

Comment:
What did you think from this approach:
```java
String targetName = "&targetFileName=" + myProzess.getTitel() + ".pdf";
int capacity = contentServerUrl.length() + targetName.length();
...
url.append(targetName); // instead of url.append("&targetName="); url.append(myProzess.getTitel()); url.append(".pdf");
```